src/components/validation/runner/telemetry.rs

```diff
@@ -37,8 +39,13 @@ impl Telemetry {
         // Attach an internal logs and internal metrics source, and send them on to a dedicated Vector
         // sink that we'll spawn a listener for to collect everything.
         let internal_logs = InternalLogsConfig::default();
-        let internal_metrics = InternalMetricsConfig::default();
-        let vector_sink = VectorSinkConfig::from_address(listen_addr.as_uri());
+        let internal_metrics = InternalMetricsConfig {
+            scrape_interval_secs: Duration::from_millis(100),
+            ..Default::default()
+        };
+        let mut vector_sink = VectorSinkConfig::from_address(listen_addr.as_uri());
+        vector_sink.batch.timeout_secs = Some(0.1);
+        vector_sink.request.retry_attempts = Some(0);
```

Comment:
Same question as before, just noting so I don't forget it's in two places.

Reply:
I added some comments about this.